spec/views/catalog/_show_sanborn_legend.erb_spec.rb

```diff
@@ -2,13 +2,13 @@
 
 require 'rails_helper'
 
-describe 'catalog/_show_sanborn_legend.html.erb', type: :view do
+describe 'catalog/_show_sanborn_legend.html.erb', js: true, type: :view do
```

Comment:
`js: true` can be removed, right? -- this was put in the feature

Reply:
Yes, good catch, @hackartisan ! Will fix...